tests/build_package.sh

```
@@ -3,9 +3,9 @@
 set -e
 
 tempdir=$(mktemp -d)
-tar -czvf $tempdir/dnsconfd-2.1.0.tar.gz --transform 's,^\./,dnsconfd-2.1.0/,' .
+tar -czvf $tempdir/dnsconfd-2.2.0.tar.gz --transform 's,^\./,dnsconfd-2.2.0/,' .
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)